Sources/KSCrashCore/include/KSCrashNamespace.h

```diff
@@ -0,0 +1,447 @@
+//
+// Auto-generated file. DO NOT EDIT!
+//
+// Generated by namespacer.py
+//
+// This header conditionally applies namespaces to all exposed symbols so that multiple instances of
+// this library can coexist in the same binary without causing duplicate symbol errors during linking.
+// Each instance will appear to be accessed the same way, such as `someFunction()`, but the
+// actual symbols in the binary will be `someFunction_mylib`, `someFunction_yourlib`, etc.
+//
+// To use this feature, define KSCRASH_NAMESPACE as whatever you'd like appended to all public symbols.
+// - Example: `KSCRASH_NAMESPACE=_mylib` will append _mylib to all public symbols.
+//
+// When public symbols are changed/added/removed, this file must be regenerated.
+// - To regenerate this file, run the Makefile command: `make namespace`
```

Comment:
Can we add CI check for that?

Reply:
OK, the clang stuff behaves VERY differently on Ubuntu, so I'll need time to debug it later. Merging for now.